src/tests/addItem.test.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * @jest-environment jsdom
+ */
+
+import TaskStore from '../taskStore.js';
+
+describe('Todo list addItem tests', () => {
+  const taskstore = new TaskStore();
+  document.body.innerHTML = '<div><ul class="todo-list"></ul></div>';
+
+  test('add Item I', () => {
+    taskstore.addTask('play football');
+    const list = document.querySelectorAll('.todo-list li');
+    expect(list).toHaveLength(1);
+  });
+  test('add Item II', () => {
+    taskstore.addTask('play tennis');
+    const list = document.querySelectorAll('.todo-list li');
+    expect(list).toHaveLength(2);
+  });
+  test('add Item III', () => {
+    taskstore.addTask('Watch fally ipupa concert');
+    const list = document.querySelectorAll('.todo-list li');
+    expect(list).toHaveLength(3);
+  });
+});
```

Comment:
- [ ] Nice work in adding the following tests, But you missed the test for `deletTask` function, kindly include it in your test cases.